### ISSUE TITLE
Side nav buttons have no accessible label

### ISSUE THREAD

**Issue by thatbudakguy** [sul-dlss/SearchWorks#3510]
We render some helpful buttons for people to jump to the top/bottom of the page, since it can get long with lots of search results.

https://github.com/sul-dlss/SearchWorks/blob/612ed54760609127c97c568ac1ad28ba1e3de530/app/views/shared/_side_nav_minimap.html.erb#L3-L6

https://github.com/sul-dlss/SearchWorks/blob/612ed54760609127c97c568ac1ad28ba1e3de530/app/views/shared/_side_nav_minimap.html.erb#L26-L29

They appear like this:

![Screenshot 2023-09-15 at 16 46 31](https://github.com/sul-dlss/SearchWorks/assets/4924494/7d95a2c6-5259-45a1-93da-d3b809dac5f4)

Because the `<span>` inside the `<button>` has some styles applied that make it `display:none;`, the screen reader is instructed to ignore it, and so it's not allowed to become the label for the button itself. As a result, the button won't be announced as anything, because it has no accessible label.

Changing the styles on the `<spans>` to Bootstrap's `.sr-only` class will keep them hidden, but remove `display:none;` so that they can serve as labels for the buttons.

This is a [WCAG Level A violation](https://www.w3.org/TR/WCAG20-TECHS/F77.html), which is something SODA has asked us to remediate whenever time allows.

[See this issue in SiteImprove](https://my2.siteimprove.com/Accessibility/1059585/NextGen/Issue/1?ruleId=12&pageSegments=&issueKind=1&tagFilterType=2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2)

**Comment by cbeer** [comment]
Is this a report from a user or from a tool?

I may be wrong about this, but this might have been intentional. These links duplicate the section headers, so aren't particularly useful for screen-reader navigation, but we reveal the label in hover/focus states if/when you happen to get there.

**Comment by thatbudakguy** [comment]
Just an automated report. It is a real problem though — an SR will still list the buttons as available to interact with, you just don't get any information at all, so it's called "button" or something else useless. You wouldn't get the text unless you had already decided to navigate to it.

In any case, it seems like non-sighted users have no use for these since they just link to the header and footer, which those users can access via landmark navigation anyway. Probably best to also add `aria-hidden="true"` to both of them and see if that makes SiteImprove happy.

**Comment by jcoyne** [comment]
Why not just add an `aria-label`?  I think the current behavior (display when tabbed to) is intentional, so this seems like the minimal solution that would fix the issue for screen readers and not affect the current behavior.

**Comment by thatbudakguy** [comment]
That would work too! It's just not adding any functionality those users don't already have, since they can navigate to the header/footer using landmarks. Either solution is fine to me.
